### DISCUSSION TITLE
shen-ping/info/kWf-7CN7BWzBJ3bBl7r7e

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#21] [Announcements]
# shen-ping/info/kWf-7CN7BWzBJ3bBl7r7e

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/info/kWf-7CN7BWzBJ3bBl7r7e?isInfo=false

**Comment by wurencaideli** [comment]
qq看点沙雕小编
